stackslib/src/chainstate/stacks/boot/signers-voting.clar

```diff
@@ -132,9 +144,14 @@
 (define-public (vote-for-aggregate-public-key (signer-index uint) (key (buff 33)) (round uint) (reward-cycle uint))
     (let ((tally-key {reward-cycle: reward-cycle, round: round, aggregate-public-key: key})
             ;; vote by signer weight
-            (signer-weight (try! (get-signer-weight signer-index reward-cycle)))
+            (signer-weight (unwrap! (get-signer-weight signer-index reward-cycle) (err ERR_GET_SIGNER_WEIGHT)))
```

Comment:
is it possible for get-signer-weight to ever return 0? i.e. for a signer to be "registered" but with 0 slots? This would probably cause errors elsewhere so I am assuming not, but figured I would ask anyway?

Reply:
There's not an explicit check that it's _not_ zero, but there is a check that the entry exists which should protect against that?